runtime/onert/core/src/ir/train/UseDefGenerator.cc

```diff
@@ -71,6 +71,29 @@ UseDefChains UseDefGenerator::operator()()
   return _training_usedefs;
 }
 
+void UseDefGenerator::visit(const train::operation::ElementwiseActivation &node)
+{
+  if (node.param().op_type != operation::ElementwiseActivation::Type::RELU)
+  {
+    throw std::runtime_error{"UseDefGenerator: Not yet supported activation type"};
+  }
+  assert(_node_to_idx.find(&node) != _node_to_idx.end());
+  const auto &op_index = _node_to_idx.at(&node);
+  const auto backwarding_op_index = TrainingOperationIndex{op_index, false};
+
+  // Insert use of forwarding output
+  const auto &out_index = node.getOutputs().at(0);
+  const auto out_forwarding_index = TrainingOperandIndex{out_index, true};
+  insertUse(out_forwarding_index, backwarding_op_index);
+
+  // Set def of backwarding(backprop) inputs
+  for (const auto &in_index : node.getInputs() | ir::Remove::UNDEFINED | ir::Remove::DUPLICATED)
+  {
+    const auto outgoing_index = TrainingOperandIndex{in_index, false};
+    insertBackPropDef(outgoing_index, backwarding_op_index);
+  }
+}
```

Comment:
(Just for curiosity + And better understanding)

I thought that `acitvation` had only one input. 

```
Input - Relu - Output
```

```
Input - Relu6 - Output
```

```
Input - Tanh - Output
``` 

Is there a reason that traversing plural 'input**s**"? 
Also Is there a specific reason for checking `UNDEFINED` and `DUPLICATED` here?

Reply:
> Is there a reason that traversing plural 'inputs"?

No, there is no reason for it. I just copied the code from other code conventionally :)

> Also Is there a specific reason for checking UNDEFINED and DUPLICATED here?

It does not have a specific reason there(ElementwiseActivation op),
But, as you know, sometimes some operations may need to check `UNDEFINED` and `DUPLICATED`. If some operations have optional inputs (e.g. bias of FC), we may need to remove it by using to check `UNDEFINED`. `DUPLICATED` doesn't seems to need any explanation. You may know it intuitively.